### PR TITLE
fix: centralise flow + reduce success screen width

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves [design inconsistencies](https://www.notion.so/safe-global/New-Transaction-flow-b15e2dc487ca406c908d0e4fb8271704#6c3f7beb913f40ea9cda25ad177f4478)

## How this PR fixes it

- Transaction status widget has been made more compact
- The close button now has a background colour]
- Success screen width has been reduced
- All transaction flow modals are centred (new transaction, flow and success screen)

## How to test it

Use the transaction creation flow, executing a transaction and observe the above.

## Screenshots

image

image

image

image

---

image

image

image

## Checklist
* [x] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/components/tx-flow/common/TxLayout/styles.module.css [hunk 1]
```diff
@@ -103,6 +104,11 @@
   margin-bottom: var(--space-2);
 }
 
+.widget {
+  /* Height of transaction type title */
+  margin-top: 46px;
```
Comment (katspaugh) [inline src/components/tx-flow/common/TxLayout/styles.module.css:109]: 👀

**Review by katspaugh — APPROVED** [review]
Thank you, sir

**Comment by katspaugh** [comment]
Merged to the RC.

File: src/components/tx-flow/common/TxLayout/styles.module.css (hunk 1 above)
Comment (iamacook) [reply src/components/tx-flow/common/TxLayout/styles.module.css:109]: I [moved the header ("transaction type title") into the main `Grid`](https://github.com/safe-global/safe-wallet-web/pull/2309/files#diff-7969ff1bbe0b04ef810b643a0721c3cc7c875b249c11ac6b2bc9b3cc65fecd36) and aligned the sidebar with the main content. Does it not align correctly on Safari?

**Pull request merged**